explorer/src/lib/components/provisioners-list/ProvisionersList.svelte

```diff
@@ -40,6 +40,11 @@
     data.key,
     calculateAdaptiveCharCount(screenWidth, 320, 1024, 4, 25)
   );
+
+  $: ownerKey = middleEllipsis(
```

Comment:
My bad about misunderstanding what should go in the tooltip, but this would be better as (replacing the bit at line 39):

```js
$: adaptiveCharCount = calculateAdaptiveCharCount(screenWidth, 320, 1024, 4, 25);
$: provisionerKey = middleEllipsis(data.key, adaptiveCharCount);
```

And then:

```svelte
<Badge
  data-tooltip-id="provisioners-tooltip"
  data-tooltip-text={ownerType === "Account"
    ? middleEllipsis(ownerValue, adaptiveCharCount)
    : ownerValue}
  text={ownerType}
/>
```

Reply:
Thanks